### PR TITLE
Todo-27-FE: Dialogs data must be removed after closing

### PR DESCRIPTION
**Pull request by itc02**

### CODE REVIEW

File: src/components/checkbox/MainCheckbox.tsx [hunk 1]
```diff
@@ -24,16 +24,17 @@ const MainCheckbox:React.FC<Props> = ({ setAllAction, clearAllAction, route  })
     }
     setIsSetAll(!isSetAll);
   }
-
+  
   useEffect(() => {
+    console.log('useEffect')
```
Comment (arpinehp) [inline src/components/checkbox/MainCheckbox.tsx]: remove?

File: src/components/dialogs/showUsers/ShowUsersDialog.tsx [hunk 2]
```diff
@@ -67,10 +67,6 @@ export const ShowUsersDialog:React.FC<Props> = ({ open, closeDialog }) => {
     setFilterCriterion(newSearchCriterion);
   }
 
-  const close = () => {
```
Comment (arpinehp) [inline src/components/dialogs/showUsers/ShowUsersDialog.tsx:70]: good clean up!

File: src/components/dialogs/addUser/AddUserDialog.tsx [hunk 3]
```diff
@@ -40,6 +43,21 @@ export const AddUserDialog:React.FC<Props> = ({ open, closeDialog }) => {
...
+    closeDialog();
+  }
+
+  const clear = (
+    setValues: (values: FormData) => void,
+    setTouched: (touched: FormikTouched<FormData>) => void,
+    setErrors: (errors: FormikErrors<FormData>) => void,
+  ) => {
+    setValues(initialValues);
+    setTouched({})
```
Comment (arpinehp) [inline src/components/dialogs/addUser/AddUserDialog.tsx]: I think you don't need this function

File: src/components/checkbox/MainCheckbox.tsx (hunk 1 above)
Comment (itc02) [reply src/components/checkbox/MainCheckbox.tsx]: yes

File: src/components/dialogs/addUser/AddUserDialog.tsx (hunk 3 above)
Comment (itc02) [reply src/components/dialogs/addUser/AddUserDialog.tsx]: I tried to remove `setTouched`, but it isn't works properly

Comment (itc02) [reply src/components/dialogs/addUser/AddUserDialog.tsx]: If I remove `setErrors` instead of `setTouched`, everything works fine.

Comment (arpinehp) [reply src/components/dialogs/addUser/AddUserDialog.tsx]: can you explain why do you use `setErrors` function and `setTouched`?

**Review by arpinehp — APPROVED** [review]
Let's merge into develop then will talk about some cases

**Pull request merged**